Tests/BugsnagEventTests.m

```diff
@@ -393,6 +393,7 @@ - (void)testHandledReportMetaData {
     NSDictionary *dict = @{@"user.handledState": [state toJson], @"user.metaData": [metadata toDictionary]};
 
     BugsnagEvent *event = [[BugsnagEvent alloc] initWithKSReport:dict];
+    [event clearMetadataFromSection:@"device"];
```

Comment:
Do we also want at least one test to check that the device property contains something?

Reply:
The existing test is asserting that the metadata dictionary has a count of 1 item after calling `addMetadata`. This assumption was invalidated because `event.metadata.device` now captures several fields that used to be on `event.device`.

As this is just a precondition for a test I'm not sure that this requires any action